src/components/Calendar/Calendar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import FullCalendar from '@fullcalendar/react';
+import dayGridPlugin from '@fullcalendar/daygrid';
+import timeGridPlugin from '@fullcalendar/timegrid';
+
+const Calendar = () => {
+    return (
+        <FullCalendar
+            plugins={[dayGridPlugin, timeGridPlugin]}
+            initialView="timeGridWeek"
+            headerToolbar={{
+                left: 'prev,next today',
+                center: 'title',
+                right: 'dayGridMonth,timeGridWeek,timeGridDay'
+              }}
+        />
+    )
+}
+
```

Comment:
Very nice basic calendar! It was fun working with you.

Reply:
Looks good so far